bundles/org.openhab.automation.jsscripting/pom.xml

```diff
@@ -122,11 +121,7 @@
       <artifactId>js-scriptengine</artifactId>
       <version>${graal.version}</version>
     </dependency>
-    <dependency>
-      <groupId>org.graalvm.js</groupId>
-      <artifactId>js-launcher</artifactId>
-      <version>${graal.version}</version>
-    </dependency>
+    <!-- org.graalvm.js/js-launcher is GraalVM's JavaScript command line interpreter and not required here -->
```

Comment:
Do we need this comment?  Its seems out of place since the config it references is not longer there.

Reply:
We don’t need this comment here 👍
I often like to have those for myself, but this one is not so important.